source/docs/3/digitizing_basics.rst

```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
-    the row for the ``Parks`` layer. 
-
-.. image:: /static/3/digitizing_basics/images/24.png
-   :align: center
-
-25. Click on :guilabel:`Add feature` to add a polygon. With the **Avoid
-    Overlap**, you will be able to quickly digitize a new
-    polygon without worrying about snapping exactly to the neighboring polygons.
-
-.. image:: /static/3/digitizing_basics/images/25.png
-   :align: center
-
-26. Right-click to finish the polygon and enter the attributes. Magically the
-    new polygon is shrunk and snapped exactly to the boundary of the neighboring
-    polygons! This is very useful when digitizing complex boundaries where you
-    need not be very precise and still have topologically correct polygon.
-    Click :guilabel:`Toggle Editing` to finish editing the ``Parks`` layer.
 
-.. image:: /static/3/digitizing_basics/images/26.png
-   :align: center
+   
+24. A pop-up dialog will appear. Select the :guilabel:`Add New Layer` button. 
 
-27. Now it is time to digitize a building's layer. Create a new polygon layer named
-    ``Buildings`` by clicking on :guilabel:`New GeoPackage Layer...` icon from Panels.
-
-.. image:: /static/3/digitizing_basics/images/27.png
-   :align: center
-
-28. Once the ``Buildings`` layer is added, turn off the ``Parks`` and ``Roads``
-    layer so the base topo map is visible. Select the ``Buildings`` layer and
-    click :guilabel:`Toggle Editing`.
-
-.. image:: /static/3/digitizing_basics/images/26.png
-   :align: center
-
-29. Digitizing buildings can be a cumbersome task. Also, it is difficult to add
-    vertices manually so that the edges are perpendicular and form a rectangle.
-    We will use a QGIS toolbar called **Shapes Digitizing** to help with
-    this task. Right-click on any empty space on the toolbar area and activate 	the ``Shapes Digitizing Toolbar``. 
-
-.. image:: /static/3/digitizing_basics/images/29.png
-   :align: center
-
-30. Zoom to an area with the buildings and click :guilabel:`Rectangle by
-    Extent` button. Click and drag the mouse to draw a perfect rectangle.
-    Similarly, add remaining buildings.
-
-.. image:: /static/3/digitizing_basics/images/30.png
-   :align: center
-
-31. You will notice that some buildings are not vertical. We will need to draw
-    a rectangle at an angle to match the building footprint. Click the
-    :guilabel:`Rectangle from center`.
+ .. image:: /static/3/digitizing_basics/images/24.png
+    :align: center   
+ 
+25. Now select layer ``Parks`` then clickroad :guilabel:`Toggle Editing` and click the :guilabel:`Add feature` button and click on the map canvas to add a polygon vertex. Digitize the polygon representing the park. Make sure you snap to the road's vertices so there are no gaps between the park polygons and road lines. Right-click to finish the polygon.
+
+ .. image:: /static/3/digitizing_basics/images/25.png
+    :align: center
+   
+26. Enter the park name in the :guilabel:`Parks - Feature Attributes` pop-up.

```

Comment:
Add another step after this showing another polygon drawn.

Reply:
Done.